deployment/terraform/strings.go

```diff
@@ -384,3 +387,18 @@ KC_DATABASE=keycloak`
 const prometheusNodeExporterConfig = `
 ARGS="--collector.ethtool"
 `
+
+const netpeekServiceFile = `
+[Unit]
+Description=netpeek
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/bin/sh -c '/usr/local/bin/netpeek -iface "$(ip route show to default | awk \'{print $5}\')" -port %d'
```

Comment:
Why do we need the `/bin/sh` here? Can't we just run the binary directly?

Reply:
It's needed to make the subcommand that finds out the default network interface work properly.

Reply:
Ah, of course! Thanks for clarifying :nerd_face: